### ISSUE TITLE
Minimal OpenSSL version ?

### ISSUE THREAD

**Issue by NotSqrt** [SoftEtherVPN/SoftEtherVPN#31]
Hi !

What is the minimal required OpenSSL version ?
I just tested on an embedded linux (with debian lenny,  openssl 0.9.8g from 2007 ... :cry: ) and I have the error:

```
-- Alert: RsaCheck() --
OpenSSL Library Init Failed. (too old?)
Please install the latest version of OpenSSL.
```

I have not found this info in the specifications, for instance.

Thanks !

**Comment by tahasiddiqui123** [comment]
still same occurred with me but still can't find the solution

**Comment by jackey** [comment]
My OpenSSL version is "OpenSSL 1.0.2a 19 Mar 2015". 
still get this error when compile from source code.

Do you have any solution ?

**Comment by gasinvein** [comment]
Same with openssl 1.0.2h

**Comment by nickdsl** [comment]
Hi! Yesterday i found the reason.
I created topic on SoftEtherVpn forum.
http://www.vpnusers.com/viewtopic.php?f=7&t=7429
Enjoy! ;)

**Comment by adminport** [comment]
https://github.com/SoftEtherVPN/SoftEtherVPN/issues/341

New maintainer needed

**Comment by moatazelmasry2** [comment]
Hi @nickdsl. Could you please create a pull request with your suggestion

**Comment by chipitsine** [comment]
can we close this issue ?

**Comment by davidebeatrici** [comment]
Fixed in #481.
